media/js/mozorg/manifesto.js

```diff
@@ -168,73 +37,34 @@ $(function() {
         window.open(url, 'twitter_share', $.param(options).replace(/&/g, ',')).focus();
     };
 
-    // Set up modal link handler
-    $(document).on('click', '#modal .principle a', function (event) {
+    // Set up link handler
+    $(document).on('click', '.principle a', function (event) {
         var $this = $(this);
-        var section = $('#modal .inner').attr('id').match(/\d+/)[0];
+        var section = $this.parents('.principle').attr('id').match(/\d+/)[0];
         var href = $this.attr('href');
         var action;
 
         if ($this.hasClass('tweet')) {
             // Open Twitter in a sub window
             event.preventDefault();
             openTwitterSubwin(section, href);
+        } else if ($this.hasClass('principle-number')) {
+            // nothing
         } else {
             // Open the link in a new tab
             $this.attr({
```

Comment:
As this page no longer has modal states, I totally wouldn't object if we killed this catch-all condition that opens *every* link in a new window. Thoughts?

Reply:
Is it okay if I leave this for the PR with the addendum in it? I want to spend some time to understand the analytics before I mess with them.